Source/Plugins/Core/com.equella.core/js/tsrc/modules/UserModule.ts

```diff
@@ -40,3 +40,16 @@ export const getCurrentUserDetails = () =>
   OEQ.LegacyContent.getCurrentUserDetails(API_BASE_URL).then(
     (result: OEQ.LegacyContent.CurrentUserDetails) => result
   );
+
+/**
+ * Lookup users known in oEQ.
+ * @param ids An array of oEQ ids
```

Comment:
We know it's an array (more or less) from the type signature. How about reworking to something more generic - and also defining which IDs:

```suggestion
 * @param ids A list of oEQ user IDs
```